<!-- DESCRITPION -->

<launch>
	<node pkg="ira_laser_tools" name="laserscan_multi_merger" type="laserscan_multi_merger" output="screen">
	<param name="destination_frame" value="mir1/bl_caster_wheel_link"/>
	<param name="cloud_destination_topic" value="/merged_cloud"/>
	<param name="scan_destination_topic" value="/scan_multi"/>
	<param name="laserscan_topics" value ="/mir1/b_scan /mir1/f_scan" /> <!-- LIST OF THE LASER SCAN TOPICS TO SUBSCRIBE -->
	<param name="angle_min" value="-2.3561899662017822"/>
	<param name="angle_max" value="2.3561899662017822"/>
	<param name="angle_increment" value="0.008726629428565502"/>
	<param name="scan_time" value="0.0"/>
	<param name="range_min" value="0.05000000074505806"/>
	<param name="range_max" value="29.0"/>
	</node>
</launch>